docs/meetings/team/2023_03_20.md

```diff
@@ -0,0 +1,68 @@
+# 🚀 Weekly Team Meeting Minutes
+
+**Start Time** 10:00pm
+
+**End Time** 10:
+
+**Location** Discord
+
+**Date** March 20, 2023
+
+## 👋 Attendence
+
+- Edouard
+- Logan
+- Matt.
+
+Attendance Notes:
+
+- Veronica is on her way back to town today!
+
+## 🃏 Joke of the Day
+
+There are two kinds of people in this world:
+
+1. Those who can extrapolate meaning from incomplete data
+
+## 📢 Announcements
+
+- This week:
+  - Peer Evaluation III due on March 24
+  - Peer Testing Report II due on March 24
+  - GitStats Week 24 due on March 24
+- Upcoming:
+  - Promo video (to show during final presentation) due April 7
+  - Final presentation slides due April 7
+  - Our Final Presentation is at 3:45 pm on April 12
+  - Wings Wednesday on April 12
+
+## 📅 Last Week (Restrospective)
+
+- How did peer testing go?
+- Results from Google Form (Veronica had one session that had 2 forms submitted, sorry!)
+- Overall went well for all of us
+- FileNotFound Error:
+  - Veronica moved back a directory and did an `npm install`
+- Character default Error:
+  - When user types in invalid avatar it does not default
```

Comment:
I believe these were the same errors

Reply:
I'm really not sure why it was produced. I tried so many different solutions and that fix was the only one that made a difference. It might've just been a fluke because I haven't had issues since.